It looks like `Templates.Test.EmptyWebTemplateTest.EmptyWebTemplateCSharp` is having issues again. `error CS1593: Delegate 'RequestDelegate' does not take 0 arguments` seems to imply the new MapGet overloads added in preview4 still not being tested again. Is this actually testing against the updated template and aspnet runtime @dotnet/aspnet-build?

```
Templates.Test.EmptyWebTemplateTest.EmptyWebTemplateCSharp
Project new web  failed to publish. Exit code 1.
/datadisks/disk1/work/A1E60924/p/dotnet-cli/dotnet publish --no-restore -c Release /bl
StdErr:
StdOut: Microsoft (R) Build Engine version 17.0.0-preview-21256-03+d07c47ade for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

/datadisks/disk1/work/A1E60924/p/dotnet-cli/sdk/6.0.100-preview.5.21264.3/MSBuild.dll -maxcpucount -property:Configuration=Release -target:Publish -verbosity:m /bl ./AspNet.bzpld23ioa3.csproj
You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
/datadisks/disk1/work/A1E60924/w/AC8C094C/e/Templates/BaseFolder/AspNet.bzpld23ioa3/Program.cs(13,17): error CS1593: Delegate 'RequestDelegate' does not take 0 arguments [/datadisks/disk1/work/A1E60924/w/AC8C094C/e/Templates/BaseFolder/AspNet.bzpld23ioa3/AspNet.bzpld23ioa3.csproj]

Expected: True
Actual:   False
```

--- a/eng/tools/GenerateFiles/Directory.Build.targets.in
+++ b/eng/tools/GenerateFiles/Directory.Build.targets.in
@@ -54,10 +54,6 @@
           Condition=" '$(IsServicingBuild)' != 'true' ">${MicrosoftAspNetCoreAppRefVersion}</TargetingPackVersion>
     </KnownFrameworkReference>
 
-    <PackageReference Include="Microsoft.Net.Compilers.Toolset"
-      Version="${MicrosoftNetCompilersToolsetVersion}"
-      PrivateAssets="all"
-      IsImplicitlyDefined="true" />
   </ItemGroup>
 
   <!-- Warn if the "just-built" ASP.NET Core shared framework does not exist. -->

--- a/eng/tools/GenerateFiles/GenerateFiles.csproj
+++ b/eng/tools/GenerateFiles/GenerateFiles.csproj
@@ -13,7 +13,6 @@
         MicrosoftAspNetCoreAppRuntimeVersion=$(SharedFxVersion);
         MicrosoftNETCoreAppRefVersion=$(MicrosoftNETCoreAppRefVersion);
         MicrosoftNETCoreAppRuntimeVersion=$(MicrosoftNETCoreAppRuntimeVersion);
-        MicrosoftNetCompilersToolsetVersion=$(MicrosoftNetCompilersToolsetVersion);
         SupportedRuntimeIdentifiers=$(SupportedRuntimeIdentifiers.Trim())
       </_TemplateProperties>
     </PropertyGroup>

